### ISSUE TITLE
[BUG] button are not accessible using screen reader or display braille

### ISSUE THREAD

**Issue by jessy836** [anthropics/claude-code#544]
## Environment
- Platform (select one):
  - [X] Anthropic API
  - [ ] AWS Bedrock
  - [ ] Google Vertex AI
  - [ ] Other: 
- Claude CLI version: 0.2.45
- Operating System: kalilinux 2025.1 on wsl2
- Terminal:  windows terminal

## Bug Description

is impossible to know what button is selected when you use a  screen reader or a display braille

## Steps to Reproduce

0. using a screen reader (e.g. nvda is free and opensource)
1. cd to/project/path
2. claude
3. ask e.g. "add a function which print hello world"
4. claude ask you if you want to apply the edits
5. is impossible to determin what button is selected using nvda

## Expected Behavior

be able to choose the option you want when using a screen reader or a display braille

## Actual Behavior

is impossible to determine what option is selected using a screen reader or a display braille

## Additional Context

some example of terminal interface accessible to screen reader and display braille
- having a prompt which allow to type the name of the option you want or it first letter
- associate a number to each options and ask to type  coresponding to the option you want to select
- suround the selected option by a symbole, for instance
  option 1
< option 2 > 
  option 3

which indicate that option 2 is selected 

thanks in advance for your time and dedication

**Comment by bcherny** [comment]
Fixed in [0.2.47](https://www.npmjs.com/package/@anthropic-ai/claude-code/v/0.2.47). Please keep the feedback coming!

**Comment by jessy836** [comment]
wow thank you for your responsiveness
